api/compliance_gcp.go

```diff
@@ -38,7 +38,7 @@ func (svc *ComplianceService) GetGcpReport(config ComplianceGcpReportConfig) (
 	err error,
 ) {
 	if config.OrganizationID == "" || config.ProjectID == "" {
-		err = errors.New("organization_id and project_id is required")
+		err = errors.New("organization_id and project_are required")
```

Comment:
?

Reply:
ugh, messed up correction. See below for what it should be